server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/scala/org/apache/james/jmap/rfc8621/contract/EmailQueryMethodContract.scala

```diff
@@ -4560,6 +4560,132 @@ trait EmailQueryMethodContract {
     }
   }
 
+  @Test
+  def inMailboxShouldBeRejectedWhenInOperator(server: GuiceJamesServer): Unit = {
```

Comment:
Why? I don't see the specs mentioning any of that?

Reply:
Because our backend don't support it.

See https://issues.apache.org/jira/browse/JAMES-3182

Reply:
Ok then. Might be nice to fix it one day if possible though